src/uct/ib/base/ib_verbs.h

```diff
@@ -96,17 +96,6 @@ static inline struct ibv_mr *ibv_exp_reg_mr(struct ibv_exp_reg_mr_in *in)
 #  define IBV_EXP_ACCESS_ALLOCATE_MR                0
 #  define IBV_EXP_HAVE_CONTIG_PAGES(_attr)          0
 
-struct ibv_exp_reg_mr_in {
```

Comment:
have you tested with older OFED versions?
Also, if you implemented instrumentation only above verbs, why do you need to disable rc_acce transport?